jax/core.py

```diff
@@ -990,6 +990,9 @@ def process_env_traces(post_processor: str, primitive: Primitive,
 
 def _call_bind(processor: str, post_processor: str, primitive: Primitive,
               f: lu.WrappedFun, *args, **params):
+  # TODO add a check like this, clean up to get rid of strings
```

Comment:
Didn't you just introduce a `map_bind` somewhere?

Reply:
Yeah, a few lines down, it's `partial(_call_bind, 'process_map', 'post_process_map')`. I should clean that up right now...

Reply:
The cleanup in #2829 makes this clearer, I think!